homeassistant/components/homekit_controller/button.py

```diff
@@ -115,3 +123,37 @@ async def async_press(self) -> None:
         key = self.entity_description.key
         val = self.entity_description.write_value
         return await self.async_put_characteristics({key: val})
+
+
+class HomeKitEcobeeClearHoldButton(CharacteristicEntity, ButtonEntity):
+    """Representation of a Button control for Ecobee clear hold request."""
+
+    def get_characteristic_types(self):
```

Comment:
Missing return typing

Reply:
The return type of this is about to change, FWIW.

Reply:
(To elaborate, when this PR is merged i'll be rebasing 3+ branches which move us towards CharacteristicsTypes and ServicesTypes being proper enums so we can use them in types here, at which point i'll be updating all the integration and tests to use them. So i'm a +0 about typing homekit_controller internals right now.)